lib/transport/helpers.ex

```diff
@@ -23,4 +27,28 @@ defmodule Transport.Helpers do
     |> String.split("/")
     |> List.last()
   end
+
+  def format_date(nil), do: ""
+  def format_date(date) do
+    with {:ok, parsed_date} <- Timex.parse(date, "{ISO:Extended}"),
+          converted_date <- Timezone.convert(parsed_date, "Europe/Paris"),
+          {:ok, formated_date} <- Formatter.format(converted_date, "{RFC3339}") do
```

Comment:
really sorry, but I think you wanted to write `formatted_date`

Reply:
pffffFFFFFFFFF